app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedFragment.kt

```diff
@@ -37,14 +36,14 @@ class FeedFragment : Fragment() {
     }
 
     private fun setupViewModel() {
-        viewModel = ViewModelProvider(this, FeedViewModelFactory())
+        viewModel = ViewModelProvider(this, FeedViewModelFactory(activity!!))
```

Comment:
```suggestion
        viewModel = ViewModelProvider(this, FeedViewModelFactory(requireContext()))
```

Reply:
Done. Yeah, that was silly to use a fingernail for a screwdriver.